python/src/tests/tasks/test_list_genes.py

```diff
@@ -82,6 +82,7 @@ def test_throws_on_missing_parameters(self):
     def test_works_with_request(self):
         ListGenes(self.correct_desc)
 
+"""
```

Comment:
same

Reply:
because we don’t have time to recreate the snapshot for the new data set. it will be commented out and i added a ticket to the ops sprint to reenable

Reply:
yes, I spoke with Oliver after that who explained me everything